### ISSUE TITLE
We forked this package and have fixed many bugs

### ISSUE THREAD

**Issue by alexeyinkin** [pd4d10/highlight#47]
This project seems abandoned, so we forked it and did the following:

- Updated to the recent HighlightJS 11.8 (was 9.18). A lot of bugs were fixed by this.
- Improved the API by adding `Language` class in addition to `Mode` to better mimic the original HighlightJS classes.
- Improved the porting script so it is easier to understand.
- Described the process here: https://medium.com/akvelon/how-we-ported-highlightjs-to-dart-82b107415fbd

The new package is published as:
1. https://pub.dev/packages/highlighting
2. https://pub.dev/packages/flutter_highlighting

We intend to maintain our fork.
See the migration guide: https://github.com/akvelon/dart-highlighting/blob/main/doc/migrating.md

*(Posting this here as there seems to be no other way to inform the users of the package who are in need of changes.)*

**Comment by pd4d10** [comment]
Hi @alexeyinkin, just see this and your previous issue https://github.com/git-touch/highlight.dart/issues/38. Thanks for the work to improve it!

> This project seems abandoned

As you said, I really don't have much time to maintain this package at the moment, so I also think it's a more appropriate choice to hand it over to a team that has enough time to maintain it.

Considering the existing users, perhaps I can add you and your coworkers as admin (for both GitHub repo and pub.dev package), and you can directly use this package name. Then existing users do not need to do additional migration work, they just need to upgrade the version.

What do you think?

**Comment by alexeyinkin** [comment]
@pd4d10 this would be awesome!

For pub.dev, please add `leha@inkin.ru` as admin of the two packages on pub.dev. I can see that they are published as 'Unverified publisher'. This means that it may be easier to do the reverse: I can add your pub.dev email as admin of my publisher, and you can transfer the packages there. If it is easier for you, please drop your email to me at my email above.

Regarding GitHub, we would rather maintain our own fork because we have a history there. What we can do in this repository is write in the README.md an encouragement to migrate and all the new links. For that, please make me an admin (or whatever it is called on GitHub).

**Comment by pd4d10** [comment]
> we would rather maintain our own fork because we have a history there

Not quite sure about what you mean, I guess a simple `git push` should sync the git history, and you and your coworkers' contributions could also be recognized.

If you're concerned about the organization name, maybe we can move this repo to a new organization, such as something like "highlight-dart", just like @highlightjs does.

What do you think?

**Comment by alexeyinkin** [comment]
OK. My employer would want it under their https://github.com/akvelon organization. Please transfer it there.

**Comment by pd4d10** [comment]
> My employer would want it under their https://github.com/akvelon organization

OK. Personally, I prefer it to be maintained by the community, but I also fully understand your request, if these work were done during the time working for the company.

Please give me some time to consider it.

Anyway, thanks very much for your work to improve it!

**Comment by alexeyinkin** [comment]
@pd4d10 can you add me on LinkedIn? https://www.linkedin.com/in/alexey-inkin-784b7371/
Communication seems to take long otherwise.
This is just to keep in touch because I no longer work for Akvelon.

**Comment by amake** [comment]
@alexeyinkin Now that you are no longer working for Akvelon, can you update us about the status of flutter_highlighting? Is there a commitment to maintain it (by who)?

**Comment by alexeyinkin** [comment]
In Akvelon, @nausharipov is now in charge of Flutter. If anyone knows the roadmap, it is him.

**Comment by busslina** [comment]
Would you adopt this PR?
https://github.com/git-touch/highlight.dart/pull/50

**Comment by nausharipov** [comment]
@busslina, yes. Thank you. I've replied here: https://github.com/akvelon/dart-highlighting/issues/74#issuecomment-1836018619

**Comment by amake** [comment]
A year later, the fork appears to be abandoned 🫤
